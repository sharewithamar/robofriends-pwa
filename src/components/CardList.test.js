import { shallow } from 'enzyme';
import React from 'react';
import CardList from './CardList';

it('expect to render Card List component', () => {
  const mockRobots = [
    { id: 1, name: 'Jon Snow', username: 'JonJon', email: 'jonsnow@gmail.com' }
  ];
  const cardComponent = shallow(<CardList robots={mockRobots} />);
  expect(cardComponent.debug()).toMatchSnapshot();
});
