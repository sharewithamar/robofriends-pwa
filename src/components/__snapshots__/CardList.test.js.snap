// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Card List component 1`] = `
"<div>
  <Card id={1} name=\\"Jon Snow\\" email=\\"jonsnow@gmail.com\\" />
</div>"
`;
